pkg/config/config.go

```diff
@@ -17,7 +17,7 @@ var (
 	// config file naming and in the profile store
 	AppName = "otdfctl"
 
-	Version   = "0.0.0"
+	Version   = "0.20.0" // x-release-please-version
```

Comment:
It looks like we set the [SEM_VER var in the workflow here](https://github.com/opentdf/otdfctl/blob/main/.github/workflows/release.yaml#L56) which is read into the `make build` step when it's [assigned to the version](https://github.com/opentdf/otdfctl/blob/main/Makefile#L5) and [set via build tag](https://github.com/opentdf/otdfctl/blob/main/Makefile#L14). 

Can you please help me see how release please updating this variable when it runs will help in xtests?

Reply:
- release please builds from source. The variable could be set there, but I'd have to recreate a lot of this logic in another repo
- other projects may include this project as a library using a go.mod dep, in which case this will always resolve to 0.0.0